server/src/assembly/resources/web/src/ServerInfo.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react'
+import './ServerInfo.css'
+
+class ServerInfo extends React.Component{
+    constructor(props) {
+        super(props);
+        this.state = {
+            list: {
+            }
+        }
+    }
+
+    componentDidMount(){
+        fetch('http://localhost:8181/rest/server_information',{
```

Comment:
Hardcode here? If I change the `http_port=8181` property in iotdb-engine.properties, can the bundle.js be built correctly?